meta-xt-control-domain-virtio/recipes-guest/domu/domu.bbappend

```diff
@@ -1,14 +1,23 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
+RDEPENDS:${PN} += "\
+bash \
```

Comment:
Hi @lorc, formatting accepted. I'll change it in both domu and doma.

As I'm using *.sh script for ExecStart, the build fails without the addition of bash dependency. It tells me that I should define RDEPENDS to bash.

Reply:
Hi @lorc, formatting done.

Reply:
It fails because your script begins with `/bin/bash` shebang. Try `/bin/sh`. 

We really want to minimize dom0 root FS image size, so we don't want to install additional shell.

Reply:
Hi @lorc, accepted. I'll try to remove the usage of the bash shell, introduced in my PR-s.